docs/topics/jobs.rst

```diff
@@ -84,3 +86,91 @@ running :class:`~scrapy.Spider` class.
 If you wish to log the requests that couldn't be serialized, you can set the
 :setting:`SCHEDULER_DEBUG` setting to ``True`` in the project's settings page.
 It is ``False`` by default.
+
+.. _jobdir-details:
+
+Job directory details
+=====================
+
+(using the DownloaderAwarePriorityQueue)::
```

Comment:
Writing this documentation will require a bit of additional research. @michael-lazar’s input is amazing, and most of the end result will probably be based on it, but we still need to iron out some details and make the information as accurate as possible for every scenario.

For example, here. Why are we mentioning the priority queue used? Does the file system structure, any file name of the content of any file change depending on the priority queue used? Is there anything else that may affect the job directory contents?

Reply:
I have to admit I have no clue about how scrapy actually works.

If you can point me to the right areas of the code I should be able to focus my attention on understanding the parts of the code that are relevant for this directory and improve the docs this way.

Reply:
You should be able to find the relevant parts searching for `JOBDIR` in the code, and following from there.

Reply:
After looking through the code, the descriptions from @michael-lazar look pretty accurate to me.
I removed some sentences that sounded "doubtful", because I think documentation should sound authoritative and not doubt itself.

I am still not sure which section need to be more elaborate to be more useful to the regular user.